entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnodeImpl.java

```diff
@@ -106,6 +156,50 @@ public MergedXnodeImpl setXnodeQ1(double xnodeQ1) {
         return this;
     }
 
+    @Override
+    public MergedXnodeImpl setB1dp(float b1dp) {
+        this.b1dp = checkDividerPosition(b1dp);
+        return this;
+    }
+
+    @Override
+    public float getB1dp() {
+        return Float.isNaN(b1dp) ? MergedXnode.super.getB1dp() : b1dp;
```

Comment:
Sorry I did not notice that! But since `checkDividerPosition` has changed, you can directly return `b1dp`

Reply:
sorry it's an oversight because made for xdp and rdp

Reply:
Sorry it's an oversight because made for xdp and rdp